### PR TITLE
feat(Sidebar): add component

### PR DESCRIPTION
**Pull request by UnbrandedTech**
Submitted passing test and structure for Sidebar

### CODE REVIEW

File: src/collections/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  static propTypes = {
+    /** An element type to render as (string or function). */
+    as: customPropTypes.as,
+
+    /** Additional classes. */
+    className: PropTypes.string,
+  }
+
+  static autoControlledProps = [
+    'visable',
```
Comment (levithomason) [inline src/collections/Sidebar/Sidebar.js:33]: Typo here `visible`.

**Comment by stubailo** [comment]
Can't wait to use this!

**Comment by SanskarSans** [comment]
@levithomason Wow waiting for this component. When shall we get to use this component?

Thanks

**Comment by levithomason** [comment]
Heads up for others, per @UnbrandedTech:

> I was going to work on it but the project I'm working on doesn't need it just yet. When my gets to the point it needs it I'll start building it out. If anyone wants to pick it up before that, feel free.

This one is up for grabs!

**Comment by levithomason** [comment]
Closing in favor of #905

**Pull request closed without merging**